### ISSUE TITLE
PHP arrays should be called array, objects called object, not dictionary

### ISSUE THREAD

**Issue by disccomp** [cadorn/legacy-fireconsole#10]
It's just an annoyance, but dictionary is not a type in PHP, and distinct types shouldn't be grouped together under dictionary.

FF 3.6.4 | FB 1.5.4 | FC 0.0.0master1002152141

**Comment by cadorn** [comment]
Yes. You are absolutely right. I have not finished the renderers for PHP yet. What you are seeing is the default renderers for the primitive objects used in encoding and transport.
